packages/js-sdk/src/sandbox/filesystem/index.ts

```diff
@@ -95,8 +80,17 @@ export class Filesystem {
     return res.data
   }
 
-  async write(path: string, data: string | ArrayBuffer | Blob | ReadableStream, opts?: FilesystemRequestOpts): Promise<EntryInfo> {
-    const blob = await new Response(data).blob()
+  async write(path: string, data: WriteData | { data: WriteData; filename: string }[], opts?: FilesystemRequestOpts): Promise<EntryInfo | EntryInfo[]> {
```

Comment:
I think we should create separate overloads for this method, taking one and multiple files because:

1. This way, you will get an array (in typing) as a return instead of an array or single entry
2. The `path` parameter that is always present right now is very confusing when you are uploading multiple files